package.json

```diff
@@ -4,11 +4,12 @@
   "description": "",
   "license": "ISC",
   "scripts": {
-    "build": "neutrino build --use neutrino-preset-node",
+    "build": "yarn lint && neutrino build --use neutrino-preset-node && cp src/package.json build && cd build && yarn",
```

Comment:
Deleting the folder makes the process more reproducible (you start from scratch every time)